packages/paste-website/stories/Pricing.stories.tsx

```diff
@@ -0,0 +1,378 @@
+      rowGap="space50"
+      maxWidth="1140px"
+    >
+      <PricingCard
+        icon={UserIcon}
+        isCurrentPlan
+        packageName="Free"
+        price="$0"
+        subheader="Per month"
+        packageList={[
+          "10 seats",
+          "2 sources",
+          "Unlimited destinations",
+          "1,000 MTU",
+          "50 Function hours",
+          "500,000 records processed for Reverse ETL",
+        ]}
+      />
+      <PricingCard
+        icon={UserIcon}
+        packageName="Team"
+        price="Starting at $120"
+        subheader="Per month"
+        cta="Try for 14 days"
+        packageList={[
+          "10 seats",
+          "2 sources",
+          "Unlimited destinations",
+          "1,000 MTU",
+          "50 Function hours",
+          "500,000 records processed for Reverse ETL",
+        ]}
+      />
+      <PricingCard
+        icon={BusinessIcon}
+        isEntity
+        packageName="Business"
+        price="Custom pricing"
+        subheader="contact sales"
+        cta="Contact us to upgrade"
+        packageList={[
+          "10 seats",
+          "unlimitted sources",
+          "Choose from 10,000 / 25,000 / 100,000 MTU",
+          "50 Function hours",
+          "1 million records processed for Reverse ETL",
+          "Monthly and annual plans available",
+        ]}
+      />
+    </Box>
+  );
+};
+
+const VisualItem = ({
+  isCurrentPlan,
+  isRecommended,
+  packageName,
+  price,
+  packageList,
+}: {
+  isCurrentPlan?: boolean;
+  isRecommended?: boolean;
+  packageName: string;
+  price: string;
+  packageList: string[];
+}): JSX.Element => {
+  return (
+    <VisualPickerRadio labelText={packageName} value={packageName}>
+      <Box>
+        <Box display="flex" columnGap="space30" justifyContent="space-between" alignItems="flex-start">
+          <Heading as="h2" variant="heading40" marginBottom="space0">
+            {packageName}
+          </Heading>
+          {isCurrentPlan ? (
+            <Badge as="span" variant="neutral">
+              Current plan
+            </Badge>
+          ) : null}
+          {isRecommended ? (
+            <Badge as="span" variant="success">
+              Recommended
+            </Badge>
+          ) : null}
+        </Box>
+        <Box marginTop="space30" display="flex" columnGap="space20" alignItems="flex-end">
+          <Heading as="h3" variant="heading10" marginBottom="space0">
+            {price}
+          </Heading>
+          <Paragraph marginBottom="space0">Per month</Paragraph>
+        </Box>
+      </Box>
+      <Box marginY="space50">
+        <Separator orientation="horizontal" />
+      </Box>
+
+      <Box display="flex" flexDirection="column" rowGap="space50">
+        {packageList.map((item: string) => (
+          <Box key={item} display="flex" columnGap="space20" alignItems="center">
+            <SelectedIcon decorative color="colorTextIconSuccess" />
+            <Paragraph marginBottom="space0">{item}</Paragraph>
```

Comment:
todo: Icon + text should align to the top when the text goes to 2 lines

Reply:
todo: when there is badge but not an icon, then the badge should keep right aligned and centered with the package name.
![Screenshot 2024-10-02 at 8 08 53 AM](https://github.com/user-attachments/assets/9362c5b1-6fd5-41f5-b3e4-bcfdd2ab6426)